styled-components/src/Text.js

```diff
@@ -0,0 +1,42 @@
+import React, { createElement as el } from 'react'
+import styled from 'styled-components'
+import {
+  color,
+  fontFamily,
+  fontSize,
+  fontWeight,
+  letterSpacing,
+  lineHeight,
+  textAlign,
+  propTypes
+} from 'styled-system'
+
+const Text = ({ is, ...props }) => {
+  const Component = styled(is)`
```

Comment:
Wanted to avoid the tagged template literal syntax, but it's seemingly required to work with `jest-styled-components`.